contrib/TestHarness/Program.cs.cmake

```diff
@@ -213,6 +214,30 @@ namespace SummarizeTest
             return ((System.Collections.IStructuralComparable)version1).CompareTo(version2, System.Collections.Generic.Comparer<int>.Default) >= 0;
         }
 
+        static bool versionLessThan(string ver1, string ver2) {
+            return !versionGreaterThanOrEqual(ver1, ver2);
+        }
+
+        static string getFdbserverVersion(string fdbserverName) {
+           using (var process = new System.Diagnostics.Process())
+           {
+              process.StartInfo.UseShellExecute = false;
+              process.StartInfo.RedirectStandardOutput = true;
+              process.StartInfo.FileName = fdbserverName;
+              process.StartInfo.Arguments = "--version";
+              process.StartInfo.RedirectStandardError = true;
+
+              process.Start();
+              var output = process.StandardOutput.ReadToEnd();
+              // If the process finished successfully, we call the parameterless WaitForExit to ensure that output buffers get flushed
+              process.WaitForExit();
+
+              var match = Regex.Match(output, @"v(\d\.\d\.\d)");

```

Comment:
This is not enough. I see some version is like `FoundationDB 6.2 (v6.2.28-PRERELEASE)`.

Reply:
The regex would match `v6.2.28`, and then the capture group would pull out `6.2.28`.  Is that wrong?

Reply:
Sounds good! I am just not sure if `\d` matches a single digit or multiple digits, i.e., some languages need `\d+`.

Reply:
Doh!  I missed your point.  You are correct.